src/mavedb/routers/hgvs.py

```diff
@@ -104,15 +111,15 @@ def list_transcripts_for_gene(gene: str, hdp: RESTDataProvider = Depends(hgvs_da
     return set([tx_info["tx_ac"] for tx_info in hdp.get_tx_for_gene(gene)])
 
 
-@router.get("/transcripts/{transcript}", status_code=200, response_model=list)
+@router.get("/transcripts/{transcript}", status_code=200)
 def transcript_info(transcript: str, hdp: RESTDataProvider = Depends(hgvs_data_provider)):
     """
     List transcript information for a particular transcript
     """
     return hdp.get_tx_identity_info(transcript)
 
 
-@router.get("/transcripts/protein/{transcript}", status_code=200, response_model=str)
```

Comment:
reminder to check this out

Reply:
yeah if this is declared with `response_model=str` then two tests fail with:
```
pydantic.error_wrappers.ValidationError: <exception str() failed>
```
when serializing the response.  This is because `hdp.get_pro_ac_for_tx_ac` can return either a string (if there's a matching protein accession) or None (if there isn't).  So instead we're returning a JSON string (eg: with quotes around it) or a JSON null.

The mavedb-ui vue function `swapNucleotideProteinAccessions` I think expects this, or at least it passes `accept: application/json` so I assume that it expects JSON back.

Reply:
Adding to the confusion, I think maybe axios, if it asks for JSON but doesn't get JSON back, just hands over the raw string received as `response.data`,